tests/codebuild/common.sh

```diff
@@ -83,7 +88,7 @@ function delete_all_resources()
   kubectl delete -n "$crd_namespace" hostingdeployment --all 
   kubectl delete -n "$crd_namespace" model --all 
 
-  force_delete_training_jobs 
+  force_delete_training_jobs "$crd_namespace"
```

Comment:
this will only work if we add a timeout for the first trainingjob delete line 82

Reply:
Why ? I can add some sleep in there.

Reply:
because each command here runs serially and would get stuck at line 82 unless there is a exit condition like `kubectl delete --wait=60sec`